front_end/ui/components/visibility.gni

```diff
@@ -14,6 +14,7 @@ default_components_visibility = [
   "../*",
   "../../../../test/unittests/front_end/ui/components/*",
   "../../../../test/unittests/front_end/panels/*",
+  "../../../../front_end/entrypoints/*",
```

Comment:
What does this do?

Reply:
It makes the module visible for the things in `devtools/front_end/entrypoints/`. We want to use the button.ts in the Gameface panel so this has to be here